src/test/java/fr/marcwrobel/jbanking/iban/RandomIbanTest.java

```diff
@@ -0,0 +1,30 @@
+package fr.marcwrobel.jbanking.iban;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.util.Random;
+import org.junit.jupiter.api.Test;
+
+class RandomIbanTest {
```

Comment:
Should the generated IBANs be valid? If so, it would be good to have a test to make sure this always passes:

```java
Iban.isValid(new RandomIban().next().toString());
```

Reply:
Being able to instantiate an `Iban` means that the provided IBAN is valid. But it's a good idea nonetheless, it will help to enforce the fact that the checks done in the `Iban` constructor are consistent with those in `isValid`.

Reply:
Test added.